weather_mv/loader_pipeline/metrics.py

```diff
@@ -45,42 +49,41 @@ def process(self,element):
         We are passing `keyed_fn=True` as we are adding a key to our element. Usually keys are added
         to later group the element by a `GroupBy` stage.
     """
+
     def decorator(func):
         @wraps(func)
         def wrapper(self, *args, **kwargs):
             # If metrics are turned off, don't do anything.
-            if (
-                not hasattr(self, 'use_metrics') or
-                (hasattr(self, 'use_metrics') and not self.use_metrics)
+            if not hasattr(self, "use_metrics") or (
+                hasattr(self, "use_metrics") and not self.use_metrics
             ):
                 for result in func(self, *args, **kwargs):
                     yield result
 
                 return
 
-            start_time = time.time()
-            time_dict = {}
+            time_dict = OrderedDict()
```

Comment:
What is the purpose of defining a empty `time_dict` here ?

Reply:
It was moot. Removed it.